### ISSUE TITLE
Stray print message prints skipped files

### ISSUE THREAD

**Issue by timheap** [timothycrosley/isort#411]
This is in released version 4.2.3. This leads to output like

```
$ isort --diff --recursive wagtail
__init__.py
__init__.py
__init__.py
... snip ...
__init__.py
__init__.py
__init__.py
Skipped 86 files
```

This seems to come from a stray print in `isort.settings.should_skip`

**Comment by timothycrosley** [comment]
Thanks for reporting, I'll fix this in 4.2.4!

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
4.2.4 has been released and should fix this issue :)

Thanks!

~Timothy
